### PR TITLE
front page hooks to xonsh-live

### PR DESCRIPTION
**Pull request by scopatz**
Hopefully my javascript is half-way decent.

### CODE REVIEW

**Comment by scopatz** [comment]
No news needed.

File: docs/index.rst [hunk 1]
```diff
@@ -56,15 +56,44 @@ that you are used to from Bash and IPython. It works on all major systems includ
...
+        border-radius: 8px;
+        position:relative;
+        float: right;
+        right: 35%;
+        bottom: 240px;
+    }
+    </style>
+    <div id="trydiv"><p style="text-align:center;">
+    <iframe id="tryframe" data-src="http://hermit.astro73.com/"
+            src="_static/xonsh-live.png" width="80%" height="480px"
```
Comment (astronouth7303) [inline docs/index.rst]: I didn't know you could do that with an iframe.

**Comment by gforsyth** [comment]
This works locally, so i guess it's just time to merge it and see what happens.

Question re: persistence -- I `touch`ed a few files and they stick around between sessions -- is there a timeout when everything gets reset?

**Comment by gforsyth** [comment]
Thanks @scopatz !  And thanks @astronouth7303 for putting together the shell-in-a-box!

**Comment by gforsyth** [comment]
Hmm... it shows up but when I click it isn't interactive, just an image that zooms in and out:
https://xonsh.github.io/xonsh-docs/dev/

**Comment by astronouth7303** [comment]
> is there a timeout when everything gets reset?

Not at this time. See xonsh/inabox

> Hmm... it shows up but when I click it isn't interactive, just an image that zooms in and out:
https://xonsh.github.io/xonsh-docs/dev/

For me, it's refusing to load the iframe because of mixed HTTP/HTTPS content.

File: docs/index.rst (hunk 1 above)
Comment (scopatz) [inline docs/index.rst]: It is pretty sneaky. I had to look it up :)

**Comment by scopatz** [comment]
Hmm how do we get around this then? https://hermit.astro73.com/ doesn't seem to work...

**Comment by astronouth7303** [comment]
No, I didn't set up HTTPS. Unfortunately, the only solution appears to be implement HTTPS.

See xonsh/inabox#1

**Comment by scopatz** [comment]
dang

**Comment by adqm** [comment]
Is there any kind of sandboxing on this?  It seems that everyone who logs in can do whatever they want (and that the user has sudo access if they can figure out the password).

I like the idea of an interactive demo, but this makes me really nervous (unless I'm missing something, it doesn't seem like there is anything in place to prevent dangerous things or misuse of those computing resources, either accidental or intentional).

**Pull request merged**